babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -236,6 +236,22 @@ SqlCreateAttribute CreateTableAttributeJournalTable() :
     { return new SqlCreateAttributeJournalTable(id, getPos()); }
 }
 
+SqlCreateAttribute CreateTableAttributeChecksum() :
+{
+    ChecksumLevel checksumLevel;
```

Comment:
checksumLevel is a bit of a weird variable name, given that it implies that there are different levels to which this option can be set. This might make sense if the possible values were LOW/MEDIUM/HIGH like they apparently used to be, but now that this has been deprecated in favor of ON/OFF/DEFAULT it might make sense to rename this to something more boolean-sounding like checksumEnabled. That said, if you are shooting for backward compatibility, then it could be fine to keep it as is.

Reply:
Yeah I think checksumEnabled would sound better since the deprecated values aren't included here.